server/controllers/user.controller.js

```diff
@@ -123,7 +140,7 @@ class UserController {
     });
     if (userResponse && userResponse.isPasswordValid(password)) {
       const { id, firstname, lastname } = userResponse;
-      const token = await generateToken({ id });
+      const token = await generateToken({ id }, '30d');
```

Comment:
What is '30d' here?

Reply:
30days?

Reply:
Any justification for making this token last 30days?

Reply:
@paradisekelech, this was initially set by @cupidcoder, we @Chiazokam and I just refactored it so we could set a short time range for password reset. it was set to 30 days based on user experience, we do not want the user to have to log in frequently